pyproject.toml

```diff
@@ -118,11 +118,14 @@ version = { attr = "_own_version_helper.version"}
 
 [tool.ruff]
 src = ["src"]
-fix = true
 lint.select = ["E", "F", "B", "UP", "YTT", "C", "DTZ", "PYI", "PT", "I", "FURB", "RUF"]
 lint.ignore = ["B028"]
 lint.preview = true
 
+[tool.ruff.lint.flake8-pytest-style]
+fixture-parentheses = true
```

Comment:
Those should be false

Reply:
It means deviating from the current style. Is it really what you want? I'll fix that.